sfn-query-large-datasets-cdk/README.md

```diff
@@ -0,0 +1,110 @@
+# Query large datasets (Amazon Athena, Amazon S3, AWS Glue, Amazon SNS)
+
+**Resources used**  
+
+| Resource Type | Name    |
+|------------------|--------------------|
+| Athena Workgroup | AthenaWorkgroup    |
+| State Machine    | AthenaStateMachine |
+| SNS Topic        | SNSTopic           |
+| S3 Bucket        | DataBucket         |
+| Glue Database    | GlueDatabase       |
+| Glue Crawler     | GlueCrawler        |  
+  
+## Start an Athena query
+
+This workflow demonstrates how to ingest a large data set in Amazon S3 and partition it through AWS Glue Crawlers, then execute Amazon Athena queries against that partition. Deploying this sample project creates an AWS Step Functions state machine, an Amazon S3 Bucket, an AWS Glue crawler, and an Amazon SNS topic.
+
+In this project, the Step Functions state machine invokes an AWS Glue crawler that partitions a large dataset in Amazon S3. Once the AWS Glue crawler returns a success message, the workflow executes Athena queries against that partition. Once query execution is successfully complete, an Amazon SNS notification is sent to an Amazon SNS topic.
```

Comment:
I'd say: "Once a query execution is successfully complete, a notification is sent through an Amazon SNS topic". Again something inherited and adding just here my personal preference.

Reply:
done & makes completely sense to me.